python/ray/tests/test_channel.py

```diff
@@ -922,11 +922,11 @@ def write(self, value):
     )
     ray.get(actor1.write.remote("hello world"))
     assert ray.get(actor1.read.remote()) == "hello world"
-    assert ray.get(actor1.read.remote()) == "hello world"
```

Comment:
why is it removed?

Reply:
`IntraProcessChannel` is now only read once, and the caching is handled by `CachedChannel`. In future we will replace `IntraProcessChannel`